libs/blocks/marquee/marquee.css

```diff
@@ -510,13 +524,20 @@
     justify-content: flex-end;
   }
 
-  .marquee.split.one-third .media.bleed {
+  html:not([dir="rtl"]) .marquee.split.one-third .media.bleed {
     position: absolute;
     width: calc(var(--grid-container-width) * 0.6667 + (100vw - var(--grid-container-width)) / 2);
     right: 0;
     height: 100%;
   }
 
+  html[dir="rtl"] .marquee.split.one-third .media.bleed {
+    position: absolute;
```

Comment:
we should not duplicate the declarations, but inherit from default rules; see previous suggestion